docs/network-drivers.md

```diff
@@ -332,3 +332,82 @@ Networks managed by HIL may span multiple switches. No special configuration
 of HIL itself is required; just register each switch as normal and ensure that
 all VLANs in the allocator's ``vlans`` option are trunked to every managed
 switch.
+
+
+## Openvswitch Driver (For development purpose only. Do not use in production.)
+
+This driver is made available so that developers can have real switch
+like functionality without having access to any real switch hardware.
+To get started you will need to:
+1.  install openvswitch in your development machine (or VM):
+
+```
+	yum install openvswitch #For fedora or Centos
+	# Following script will:
+	# * enable the openvswitch service;
+	# * Start the openvswitch server;
+	# * Show the status to the user.
+	
+	do service openvswitch enable.
+	do service openvswitch start.
+	do service openvswitch status.
+	ovs-vsctl show
+```
+
+2.  create a bridge and name it <switch_name>:
+
+```
+	ovs-vsctl add-br <switch_name>
+```
+
+
+3.  Add some ports to this bridge:
+
+```
+	ovs-vsctl add-port <switch_name> <port_name>
+```
+
+These are just illustrative examples. You may have to do more to setup
+your switch before using it with HIL.
+
+
+Optionally, a script is made available for reference at::
+
+	https://github.com/SahilTikale/HIL_contrib/blob/master/hilInYourLap/create_datacenter.sh
+
+**Warning**: Use the script at your own discretion.
+
+
+
+To register the driver with HIL, you will need the openvswitch bridge name,
+  username of your machine; sudo password of this user.
+
+#### switch_register
+
+To register an openvswitch, the ``"type"`` field of the request body
+must have a value of::
+
+        http://schema.massopencloud.org/haas/v0/switches/ovs
+
+In addition, it requires three extra fields:
+``"username"``, ``"hostname"``, and ``"password"``, which provide the necessary
+information to connect to the openvswitch.
+``"hostname"`` has to be the name of the bridge created in step 2.
+``"username"`` is the username for your development machine (or VM)
+``"password"`` is the sudo password of your machine (or VM).
+
```

Comment:
This section needs to be updated since now that we don't require those additional fields.